src/treelearner/serial_tree_learner.cpp

```diff
@@ -702,9 +702,6 @@ void SerialTreeLearner::ComputeBestSplitForFeature(
     FeatureHistogram* histogram_array_, int feature_index, int real_fidx,
     bool is_feature_used, int num_data, const LeafSplits* leaf_splits,
     SplitInfo* best_split) {
-  if (!is_feature_used) {
-    return;
-  }
   SplitInfo new_split;
   double parent_output;
   if (leaf_splits->leaf_index() == 0) {
```

Comment:
@btrotta it seems the parent output is calculated repeatedly here for root node. I think this cost could be reduced.

Reply:
Ok, I'll take a look